sapp/tests/warning_messages_test.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import json
+import pathlib
+import tempfile
+from contextlib import ExitStack
+from unittest import TestCase
+
+from sqlalchemy import Table
+
+from ..db import DB, DBType
+from ..models import WarningMessage, create as create_model
+from ..warning_messages import update_warning_messages, upsert_entry
+
+test_metadata = {
+    "codes": {
+        "1001": "Updated warning message from test",
+        "1002": "Updated warning message from test 2",
+    },
+    "codes2": {
+        "2001": "Updated warning message from another test",
+        "2002": "Updated warning message from another test 2",
+    },
+}
+
+
+class WarningMessagesTest(TestCase):
+    def setUp(self) -> None:
+        self.db = DB(DBType.MEMORY)
+        create_model(self.db)
+
+    def test_update_warning_messages(self) -> None:
+        with ExitStack() as stack:
+            temp = stack.enter_context(tempfile.NamedTemporaryFile(mode="w+"))
```

Comment:
nitpick: use a better name for this. We try to avoid abbreviations in our codebase.

Reply:
Will do that.